testsuite/integration-arquillian/pom.xml

```diff
@@ -320,6 +320,157 @@
                 </pluginManagement>
             </build>
         </profile>
+        <profile>
+            <id>mysql</id>
+            <properties>
+                <keycloak.connectionsJpa.database>keycloak</keycloak.connectionsJpa.database>
+                <keycloak.connectionsJpa.user>keycloak</keycloak.connectionsJpa.user>
+                <keycloak.connectionsJpa.password>keycloak</keycloak.connectionsJpa.password>
+                <keycloak.connectionsJpa.url>jdbc:mysql://${auth.server.db.host}/${keycloak.connectionsJpa.database}</keycloak.connectionsJpa.url>
+                <jdbc.mvn.groupId>mysql</jdbc.mvn.groupId>
+                <jdbc.mvn.artifactId>mysql-connector-java</jdbc.mvn.artifactId>
+                <jdbc.mvn.version>5.1.29</jdbc.mvn.version>
```

Comment:
This should be `${mysql.version}`

Reply:
"JDBC driver version" is the meaning of the `${mysql.version}`.

Reply:
I don't think the name is good. Besides, the `jdbc.mvn.version` is already in use by our testsuite to properly download and install the driver.

I tried to reuse what we already have in the testsuite as much as possible.

Reply:
We have the same intention :) ${mysql.version} is already available (same for postgres), and I agree the name is unfortunate but it's already there: see e.g.
https://github.com/keycloak/keycloak/blob/master/pom.xml#L112
https://github.com/keycloak/keycloak/blob/master/testsuite/jetty/jetty93/pom.xml#L346

Reply:
Now I see what you mean, sorry. You don't want to change the property name but get the value from mysql.version.

Sometimes I get stuck with my own thoughts :)